src/chaat/db/friend_request.clj

```diff
@@ -0,0 +1,60 @@
+(ns chaat.db.friend-request
+  (:require
+   [next.jdbc :as jdbc]
+   [next.jdbc.sql :as sql]
+   [chaat.handler.errors :refer [error-table]]
+   [chaat.db.user :as user]
+   [java-time.api :as jt]))
+
+;; Will be more logical to have the user-exists? and exists? check in the model layer.
+;; Same for the friend request exists? check
+(defn exists?
+  "Check if friend request id exists in friend_requests table"
+  [connection id]
+  (not (empty? (sql/find-by-keys connection :friend_requests {:id id}))))
+
+(defn insert
+  "Insert a friend request into the db"
+  [db {:keys [sender_username recipient_username] :as content}]
```

Comment:
`content` could be renamed to `friend-request`

Reply:
It's usually worth doing the hypen <> underscore translation at the db boundary to make the rest of your clojure code consistent.